model-optimizer/extensions/front/instance_normalization.py

```diff
@@ -29,18 +35,26 @@ class InstanceNormalization(FrontReplacementOp):
     enabled = True
 
     def replace_op(self, graph: Graph, node: Node):
-
-        # Add new nodes
-        mvn = MVN(graph, {'eps': node.epsilon, 'name': node.name + '/Ins_Norm/MVN_', }).create_node()
-        mul = Mul(graph, {'axis': 1, 'name': node.name + '/Ins_Norm/mul_'}).create_node()
-        add = Add(graph, {'axis': 1, 'name': node.name + '/Ins_Norm/add_'}).create_node()
-
-        # Connect nodes
+        name = node.soft_get('name', node.id)
+
+        # create range of axes for MVN based on `start_axis` and rank of input
+        rank = Rank(graph, {'name': name + '/Rank'}).create_node()
+        rng = create_op_with_const_inputs(graph, Range, {0: int64_array(2), 2: int64_array(1)},
+                                          {'name': name + '/Range', 'output_type': np.int64})
+        mvn = MVN(graph, {'eps': node.epsilon, 'eps_mode': 'inside_sqrt', 'normalize_variance': 1,
```

Comment:
nit: remove the trailing "_"  character in the node names in the transformations you add. Since you generate the name based on the input node name the final node name will be unique, so this trailing "_" looks redundant